src/frontend/dash/renderCircles.ts

```diff
@@ -3,18 +3,23 @@ import type { MemberCircle } from "./circlePacker"
 
 const membersDiv = document.getElementById('members')
 
+
 const BUBBLE_COLORS = ['rgba(35,132,198,.5)', 'rgba(255,214,0,.5)', 'rgba(241,93,34,.5)', 'rgba(108,157,204,.5)']
 function renderCircle(circle: MemberCircle) {
     const { maxX, maxY, minX, minY } = getBounds()
-    const maxLength = Math.max(maxX - minX, maxY - minY)
+
+    const widthMultiplier = window.outerWidth * 0.4750078125 / membersDiv.clientWidth
+    const heightMultiplier = window.outerHeight * 0.7710505952380952 / membersDiv.clientHeight
+
+    const maxLength = Math.max((maxX - minX) * widthMultiplier, (maxY - minY) *  heightMultiplier)
```

Comment:
What's the point of maxLength

Reply:
maxLength was already there

Reply:
Then why did you write it? Did you just delete and rewrite it?

Reply:
Yes

Reply:
If it's not used, may as well delete it
```suggestion
```